src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+
+import Duke.TaskList;
+import Exceptions.DukeMainExceptions;
+import Duke.Storage.Storage;
+import Duke.Ui;
+import Duke.Parser;
+
+import Exceptions.CommandNotFoundException;
+
+import java.io.IOException;
```

Comment:
Your import orders are slightly off due to not being in alphabetical order, and the java.io imports should be above the duke imports!